src/state/digital-exchange/actions.js

```diff
@@ -83,20 +90,45 @@ export const filterByRating = (rating, paginationMetadata) => {
   return applyFilter(filter, paginationMetadata);
 };
 
+export const resetSearchFilter = paginationMetadata => (dispatch, getState) => {
+  const selectedCategory = getSelectedDECategory(getState());
+  dispatch(setDEFilter(
+    { formValues: { name: null }, operators: { name: null } },
+    selectedCategory,
+  ));
+  dispatch(setDEFilter(
+    { formValues: { description: null }, operators: { description: null } },
+    selectedCategory,
+  ));
+  dispatch(setDEFilter({ formValues: { id: null }, operators: { id: null } }, selectedCategory));
+  dispatch(setDEFilter(
+    { formValues: { version: null }, operators: { version: null } },
+    selectedCategory,
+  ));
```

Comment:
I think it's better to create a redux action to clear search values (maybe like a `clearDESearchFilter` method or something). This should ease up this part here